client/app/bundles/HelloWorld/startup/registration.jsx

```diff
@@ -0,0 +1,14 @@
+import ReactOnRails from 'react-on-rails';
+
+import HelloWorld from '../components/HelloWorld';
+
+import Header from '../components/Header';
+
+import ResturantsList from '../components/ResturantsList';
+
+import ResturantItem from '../components/ResturantItem';
+
+// This is how react_on_rails can see the HelloWorld in the browser.
+ReactOnRails.register({
+  HelloWorld, Header, ResturantsList, ResturantItem
```

Comment:
A good practice would be to wrap the entire application in a `<App />` component / page so you'd have full control over it. Later on, it would be useful for adding `browserHistory` and state-management like Redux.